src/api/firebase.js

```diff
@@ -165,6 +165,19 @@ export async function addItem(listPath, { itemName, daysUntilNextPurchase }) {
 	const listCollectionRef = collection(db, listPath, 'items');
 	// TODO: Replace this call to console.log with the appropriate
 	// Firebase function, so this information is sent to your database!
+
+	//This saves item to database.
+	await setDoc(listCollectionRef, {
```

Comment:
I do think the `await setDoc` is correct. I know we were looking at when creating a list how it works. I think we can use the console.log in the return to build the info for the `setDoc` here at this call. 

```suggestion
await setDoc(listCollectionRef, {
		dateCreated: new Date(),
		// NOTE: This is null because the item has just been created.
		// We'll use updateItem to put a Date here when the item is purchased!
		dateLastPurchased: null,
		dateNextPurchased: getFutureDate(parseInt(daysUntilNextPurchase)),
		name: itemName,
		totalPurchases: 0,
	});
```

Reply:
I believe this will be `await addDoc` to get it to add!